datafusion/sql/src/planner.rs

```diff
@@ -362,7 +362,7 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                 let data_type = self.convert_data_type(inner_sql_type)?;
 
                 Ok(DataType::List(Arc::new(Field::new(
-                    "field", data_type, true,
+                    "item", data_type, true,
```

Comment:
Maybe we could use `DataType::new_list_field()` https://docs.rs/arrow/latest/arrow/datatypes/struct.Field.html#method.new_list_field

Reply:
Updated. Thank you @alamb , I didn't notice this convenient API.